platform/linux-generic/odp_schedule_basic.c

```diff
@@ -22,7 +22,7 @@
 #include <odp/api/plat/thread_inlines.h>
 #include <odp/api/time.h>
 #include <odp/api/plat/time_inlines.h>
-#include <odp/api/spinlock.h>
+#include <odp/api/ticketlock.h>
```

Comment:
Commit message: `Ticket lock should may give ...`

Reply:
fixed in v6